src/util/path/index.ts

```diff
@@ -847,41 +850,31 @@ export const parsePath = (pathString: string): TComplexPathData => {
   pathString = cleanupSvgAttribute(pathString);
 
   const res: TComplexPathData = [];
-  for (const match of pathString.matchAll(new RegExp(rePathCommand, 'gi'))) {
-    let matchStr = match[0];
+  for (let [match] of pathString.matchAll(rePathCmdAll)) {
     const chain: TComplexPathData = [];
     let paramArr: RegExpExecArray | null;
     do {
-      paramArr = new RegExp(rePathCommand, 'i').exec(matchStr);
+      paramArr = rePathCmd.exec(match);
       if (!paramArr) {
         break;
       }
       // ignore undefined match groups
-      const filteredGroups = paramArr.filter((g) => g);
+      const filteredGroups = paramArr.filter(Boolean);
```

Comment:
This equal to call Boolean as `Boolean(g, index, paramArr)` this is usually suggested to avoid, i would aboid

Reply:
I'd avoid `filter(Boolean)` also because from TS 5.5 it will be able to automatically type guard `.filter(g => !!g)` to defined values, but `.filter(Boolean)` won't work